vllm/tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import copy
+
+import pytest
+
+from datadog_checks.dev import docker_run
+from datadog_checks.dev.conditions import CheckDockerLogs, CheckEndpoints
+
+from .common import COMPOSE_FILE, MOCKED_INSTANCE, MOCKED_VERSION_ENDPOINT
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
```

Comment:
Not sure where the best place is for this, but we should explain why we don't have a working vLLM instance for our tests. That's the place where we mention the GPU and CPU restrictions that stopped us.

I guess this fixture is a good place for that. Separate readme section on "testing" is another good place.